pkg/apis/pingcap/v1alpha1/tidbcluster.go

```diff
@@ -403,3 +404,14 @@ func (tc *TidbCluster) Timezone() string {
 	}
 	return tz
 }
+
+func (tc *TidbCluster) GetInstanceName() string {
```

Comment:
Alternatively, we could define a `GetLabels()` method and shadow the one of  `ObjectMeta` so that the controller logic don't have to be modified, but that way we have to copy a map every time this method get called.